docs/Changelog.md

```diff
@@ -9451,6 +9451,83 @@ This version of the operator has been available since version 9 of the default O
 </dl>
 
 ## Version 10 of the default ONNX operator set
+### <a name="Reverse-10"></a>**Reverse-10**</a>
+
+  Reverse a tensor of arbitrary shape along the given 'axes' in its dimensions.
+  
+  Example 1:
+    input = [
+      [0.0, 1.0, 2.0],
+      [3.0, 4.0, 5.0],
+    ]
+  
+    output = [
+      [5.0, 4.0, 3.0],
+      [2.0, 1.0, 0.0],
+    ]
+  
+  Example 2:
+    input = [
+      [0.0, 1.0, 2.0],
+      [3.0, 4.0, 5.0],
+    ]
+  
+    axes = [0]
+  
+    output = [
+      [3.0, 4.0, 5.0],
+      [0.0, 1.0, 2.0],
+    ]
+    
+  Example 3:
+    input = [
+      [[0.0, 1.0, 2.0],
+       [3.0, 4.0, 5.0]],
+      [[6.0, 7.0, 8.0],
+       [9.0, 10.0, 11.0]],
+    ]
+  
+    axes = [1,-1]
+  
+    output = [
+      [[5.0, 4.0, 3.0],
+       [2.0, 1.0, 0.0]],
+      [[11.0, 10.0, 9.0],
+       [8.0, 7.0, 6.0]],
+    ]
+
+#### Version
+
+This version of the operator has been available since version 10 of the default ONNX operator set.
+
+#### Attributes
+
+<dl>
+<dt><tt>axes</tt> : list of ints</dt>
+<dd>A list of integers indicating the axes to reverse. Negative values mean counting dimensions from the back.By default, reverse over all axes of input dimensions</dd>
```

Comment:
Sorry for this clarification so late on - but I assume this operator doesn't support "implicit" copying of Tensors by specifying duplicates in axes ?  For example, would axes of [0,0] mean no reversing but force the tensor's buffer to be copied ? Or simply, don't allow duplicates in axes at all - just like the way tf operates - https://www.tensorflow.org/api_docs/python/tf/reverse ?

Reply:
Great question, this is more in line with the latter, no duplicate is expected here.